src/components/TextButton.js

```diff
@@ -1,26 +1,28 @@
 import styled from 'styled-components';
 
+import { AP_THEME } from '../containers/App/themeConstants';
+
 const TextButton = styled.button`
   margin: 0;
   padding: 0;
   border: 0;
   background: transparent;
   font-size: ${props => (props.fontSize ? props.fontSize : '#inherit')};
```

Comment:
the `inherit` value should not have a `#`

Reply:
Caught yourself! https://github.com/accordproject/template-studio-v2/commit/30c061d81042410296a1852920a141bb4867b6af#diff-80fa4b1ac52713917acfa4eedc30ee5eR8